### PR TITLE
[Docs]Fix Learning Rate Finder ref name

### PR DESCRIPTION
**Pull request by xyangk**
## What does this PR do?

Fixes #\<issue_number>

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/starter/introduction.rst [hunk 1]
```diff
@@ -584,7 +584,7 @@ Once you define and train your first Lightning model, you might want to try othe
 - :doc:`Automatic early stopping <../common/early_stopping>`
 - :ref:`Automatic truncated-back-propagation-through-time <common/lightning_module:truncated_bptt_steps>`
 - :ref:`Automatically scale your batch size <advanced/training_tricks:Batch Size Finder>`
-- :ref:`Automatically scale your batch size <advanced/training_tricks:Learning Rate Finder>`
+- :ref:`Automatically scale your learning rate <advanced/training_tricks:Learning Rate Finder>`
```
Comment (rohitgr7) [inline docs/source/starter/introduction.rst:587]:
```suggestion
- :ref:`Automatically find an optimal learning rate <advanced/training_tricks:Learning Rate Finder>`
```

**Review by rohitgr7 — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by kaushikb11 — APPROVED** [review]

**Comment by rohitgr7** [comment]
looks like this part is updated on master. Closing this.
